### PR TITLE
Update CHANGELOG and pyproject.toml to 1.7.0

### PR DESCRIPTION
**Pull request by JST5000**
## High Level Overview of Change

Title says it all

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,18 +6,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+## [1.7.0] - 2022-10-12
 ### Added:
 - Support for ExpandedSignerList amendment that expands the maximum signer list to 32 entries
 - Function to parse the final account balances from a transaction's metadata
 - Function to parse order book changes from a transaction's metadata
 - Support for Ed25519 seeds that don't use the `sEd` prefix
 - Add docs to`get_account_transactions` explaining how to allow pagination through all transaction history [#462]
 - Common field `ticket_sequence` to Transaction class
+- Tests can now be run with an easier to remember command - `poetry run poe test_unit`
```
Comment (mvadari) [inline CHANGELOG.md:18]: This doesn't need to be in the changelog, since it's not relevant for non-contributors

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,18 +6,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
 ### Fixed:
 - Typing for factory classmethods on models
 - Use properly encoded transactions in `Sign`, `SignFor`, and `SignAndSubmit`
 - Fix Sphinx build errors due to incompatible version bumps
+- AuthAccount models no longer need their `account` paramter to be capitalized (it was mismatched with rippled)
```
Comment (mvadari) [inline CHANGELOG.md]: Isn't this an AMM-specific thing?

Comment (JST5000) [reply CHANGELOG.md]: You are correct - sorry I got confused with the two branches (I just saw the PR was merged and forgot there was a separate beta branch)

Comment (mvadari) [reply CHANGELOG.md]: If you're just trying to see the PRs that have been added to the master branch, you can do something like this: https://github.com/XRPLF/xrpl-py/compare/v1.6.0...master

**Review by connorjchen — APPROVED** [review]
LGTM

**Pull request merged**